integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This directory contains integration tests that verify the system components work together correctly.
+
+## Prerequisites
+
+- Node, Docker an NPM installed
+
+## Setup environment
+
+In order to run the integration tests, you need components to integrate with: DB, Clearnode (BE) and blockchain node. All of these three components can be run using Docker.
+
+Go to the root of the project (`..` from this file) and run:
+
+```bash
+docker-compose up -Vd
+```
+
+In will build clearnode from the current version of the code (you could use docker image instead, but it may not include the latest changes). Compile smart contracts and deploy them on predetermined addresses (based on mnemonic and unique salt):
```

Comment:
Please specify that `salt` is a number specified in parenthesis after the contract name.